src/main/java/org/example/models/Employee.java

```diff
@@ -0,0 +1,63 @@
+package org.example.models;
+
+//abstract class since no Employee objects are ever created
+public abstract class Employee {
+
+    private int id;
+    private String name;
+    private double salary;
+    private String bankNumber;
+    private String nationalInsuranceNumber;
+
+
+    public String getNationalInsuranceNumber() {
+        return nationalInsuranceNumber;
+    }
+
+    public void setNationalInsuranceNumber(String nationalInsuranceNumber) {
+        //Here we can put validation code to ensure the insurance number is in correct format
+        //first two characters are letters, following 6 are digits, last is a letter
+        //total length == 9
+        this.nationalInsuranceNumber = nationalInsuranceNumber;
+    }
+
+    public String getBankNumber() {
+        return bankNumber;
+    }
+
+    public void setBankNumber(String bankNumber) {
+        //validation code, ensuring it meets at least one international bank number format
```

Comment:
Minor: you can add todo's `//TODO: add validation code to ensure bank number format met` instead and it will be more obvious future functionality is coming/expected. Comments tend to often go unread or forgotten about during cleanup.